### PR TITLE
1818 add local-preview command

### PR DESCRIPTION
**Pull request by therealmarv**
- [ ] https://github.com/openstax/ce/issues/1818

### CODE REVIEW

File: test/test-step-03.bash [hunk 1]
```diff
@@ -13,4 +13,20 @@ KCOV_DIR=_kcov03 \
...
+
+
+# Check local-preview works
+SKIP_DOCKER_BUILD=1 \
+KCOV_DIR=_kcov03 \
+../enki --keep-data --data-dir $BOOK_DIR --command local-preview
+
+# Check local-preview directories / files existing
+LOCAL_PREVIEW="$BOOK_DIR/local-preview"
+[ -d "$LOCAL_PREVIEW" ] || echo "local-preview: Directory $LOCAL_PREVIEW missing"
```
Comment (TylerZeroMaster) [inline test/test-step-03.bash:26]: Should there be a non-zero exit code if one of these conditions is not met?

Comment (therealmarv) [reply test/test-step-03.bash:26]: oh, big mistake here 🤦‍♂️ will fix

**Comment by therealmarv** [comment]
Testplan:

Follow `local-preview` commands in README [here](https://github.com/openstax/enki/blob/bc4e3ea1a6260424df954aeb9648b99bb0c80e41/README.md#local-preview-of-html-files)

and open the resulting HTML files in a browser directly. The test is successful if the sample image (the duck with the hat) is visible.

**Comment by omehes** [comment]
testing: followed steps in the link above

osbooks-otto-book
slug="Tschüß"

./enki --command all-git-web --data-dir ./data/otto/ --repo "$repo" --book-slug "$slug" --ref main --style default
./enki --command local-preview --data-dir ./data/otto/

0CB4A200-BA87-4934-92F7-F9CAB6252FA3
FB82E538-893A-42C1-A122-83388E9F264D

**Pull request merged**